demos/using-dev-build/README.md

```diff
@@ -31,6 +31,42 @@ At this point, the replica (for all practical matters, a local version of the In
 
 If the IC actually lets the call (request) through to the `whoami` canister, it means that everything checked out, and the `whoami` canister just responds with the information the IC adds to requests, namely your identity (principal).
 
+### Using the Auth-Client Library To Log In With Internet Identity
+
+DFINITY provides an easy-to-use library to log in with Internet Identity: [agent-js](https://github.com/dfinity/agent-js)
```

Comment:
Here and below "(... is available here: [client auth protocol specification](#))" links are given as "this is the document: link". In the rest of the document I tried to include the links in the flow of the text to prevent it looking like a list of links; can something similar be done here?